<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="omgindia244" name="com.excel.utility.ExcelReader" tests="1" failures="0" timestamp="19 Sep 2019 08:53:56 GMT" time="0.027" errors="1">
  <testcase name="test" time="0.027" classname="com.excel.utility.ExcelReader">
    <error type="java.io.FileNotFoundException" message="C:\Users\ajeet.yadav\Desktop\TestData.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ajeet.yadav\Desktop\TestData.xlsx (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at com.excel.utility.ExcelReader.exceldata(ExcelReader.java:19)
	at com.excel.utility.ExcelReader.test(ExcelReader.java:72)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- test -->
</testsuite> <!-- com.excel.utility.ExcelReader -->
